src/components/Calculator.js

```diff
@@ -0,0 +1,65 @@
+const Calculator = () => {
+  return (
+    <div className="keyButtons">
+      <button data="AC" className="btn" type="button">
+        AC
+      </button>
+      <button data="+/-" className="btn" type="button">
+        +/-
+      </button>
+      <button data="%" className="btn" type="button">
+        %
+      </button>
```

Comment:
- [ ] Please ensure that the `Calculator` component has at least 1 child component as per requirements 👍